connectors/es/sink.py

```diff
@@ -129,10 +133,22 @@ def _bulk_op(self, doc, operation=OP_INDEX):
                 {"doc": doc["doc"], "doc_as_upsert": True},
             ]
         if operation == OP_DELETE:
+            self._log_deleted_doc_id_if_enabled(doc_id)
+
             return [{operation: {"_index": index, "_id": doc_id}}]
 
         raise TypeError(operation)
 
+    def _log_deleted_doc_id_if_enabled(self, doc_id):
+        if self._log_deleted_doc_ids:
+            msg = f"Deleted document with id '{doc_id}'"
+
+            if self._log_deleted_doc_ids_log_level == "INFO":
+                self._logger.info(msg)
+
+            elif self._log_deleted_doc_ids_log_level == "DEBUG":
+                self._logger.debug(msg)
```

Comment:
Looks like the only options are `INFO` and `DEBUG`. Should probably state that in the comment in `config.yml` and raise an error or default to something if the config has a value other than one of these.

Reply:
I'm not sure if change is needed ATM - it was an ask of a customer at some point and they found a way around it.

If it's needed, we can probably approach it a bit more generically by logging added/updated/deleted document ids in DEBUG log level.

But again, I'm not sure if it's needed any more.

Reply:
> Looks like the only options are INFO and DEBUG. Should probably state that in the comment in config.yml and raise an error or default to something if the config has a value other than one of these.

Good point, will address it!

> But again, I'm not sure if it's needed any more.
Checking it with product